mkosi/distributions/debian.py

```diff
@@ -172,18 +172,32 @@ def setup_apt(state: MkosiState, repos: Sequence[str]) -> None:
     state.root.joinpath("var/lib/dpkg").mkdir(mode=0o755, exist_ok=True)
     state.root.joinpath("var/lib/dpkg/status").touch()
 
-    config = state.pkgmngr / "etc/apt/apt.conf"
+    # We have a special apt.conf outside of pkgmngr dir that only configures "Dir" that we pass to
+    # APT_CONFIG to tell apt it should read config files in pkgmngr instead of in its usual locations. This
+    # is required because apt parses CLI configuration options after parsing its configuration files and as
+    # such we can't use CLI options to tell apt where to look for configuration files.
+    config = state.workspace / "apt.conf"
+    if not config.exists():
```

Comment:
Should this file not always be overwritten?

Reply:
Why? If it exists, it'll have the content we want. Its wasteful to write the same thing over and over again. It's in the workspace so the user can't control the content.

Reply:
True, nvm